### DISCUSSION TITLE
Thanks for creating this!

### DISCUSSION THREAD

**Discussion by Michael-F-Ellis** [janpfeifer/gonb#95] [General]
A few years ago I more or less abandoned Python for Go. One of the few things I missed was the ability to use iPython as a quick way to test how certain functions worked.  Sure, there was always the Go Playground, but you can't do file i/o with that.  I also tried `yaegi` but found it really incovenient.

I've only been playing with `gonb` for a couple of days, but it seems like a exactly what I've been looking for. 
 It's a definite productivity boost - especially in combination with VsCode + GitHub Copilot.

**Comment by janpfeifer** [comment]
hey @Michael-F-Ellis , thanks for message, it's really appreciated.

While I'm of course a big user of `gonb` -- both for quick programs, but also to test libraries/program I'm developing -- but it wouldn't have been worth it if others didn't use it either :)

Let me know if you need any feature, I'm slowly but steadily always trying to improve it.
